### PR TITLE
Doc/add ts fsrs workflow

### PR DESCRIPTION
**Pull request by ishiko732**
public link:
- draw.io : [ts-fsrs-workflow.drawio](https://app.diagrams.net/?tags=%7B%7D&lightbox=1&highlight=0000ff&edit=_blank&layers=1&nav=1&title=ts-fsrs-workflow.drawio#R%3Cmxfile%3E%3Cdiagram%20id%3D%22aEf3h9zvOVPK_aX-VWGx%22%20name%3D%22TS-FSRS%20WorkFlow%22%3E7V1bc5s4FP4t%2B%2BCZZmec4X55TJyk3ZludyfembZPGRmErQkgr8CJ01%2B%2FEggMsoixDdjtOmkSc4SEpPOdqyQ60ifR%2BiMBy8Wf2IfhSFP89Ui%2FG2maaugG%2FcMobznF1p2cMCfI5zdtCFP0A3Kiwqkr5MOkdmOKcZiiZZ3o4TiGXlqjAULwa%2F22AIf1py7BHG4Rph4It6lfkZ8ucqqj2Rv6J4jmi%2BLJquXmJREobuYjSRbAx68Vkn4%2F0icE4zT%2FFK0nMGSTV8xLXu%2BhobTsGIFx2qbC4o%2Fv5vxj9OhbT%2FfP02%2BfPv9YgzFv5QWEKz7gkWaFtL3bANNmaa%2FTNz4V1r8rXBSMk4xRN%2FQG1VpSZt9uyumnOf%2BbNTQrCP9Mxw%2FTx2lBpz2dbe5VvmLyHIR0gjbFeReKO7RabzQ6nUv20VvN6J%2Fb1wVK4XQJPEZ7pViktEUahfRKZZ3Aq9iH%2FudZSQDe85ww6l%2BrNEQx5HQfkOe%2FaDMoZYBVrhWzTtQyqlr2qDr%2FxWRCksJ1hcT58RHiCKbkjd7CS02dw4gLh6FzrLxuoKbZnLaowMzhNMDRPS%2Bb3gCAfuAY2AMP2oB4%2BHL%2FVY6FdgCgo6FKoAXvO2BUIcOcT5a5zSfVGJJPeiOfZkcx6TMEJEbxnPXuHd7MiISrRzz2EW4evM9TW4PSaHjw%2FXp0%2FzC6nYzcGzWiN5rsl6pE5whCQ6lrC8s4NQqNRhQmSxB3qy0e4QuCrzsURv7YM%2BSV65yaV9aAmv1mDlB8jG4nubnm%2FNjBrgCF4QSHmGR19SAwXYvRk5TgZ1iUxJjZ%2BE6stlvnrWlv81ZmtM2%2BWGsPKIafAPGPEcLjODsLFKVHzlr2mXHWGVBoP2K8i7O9yayizMwg6JGztnlmnFVltlOYUW9FXsoJhbF%2Fw%2BJZFvKEIEmQV5%2FRzfQrWcSSLCpVaY%2B%2F0YsxjWdUs6B8z0IZ3dUKwt2a186v3oqrNUq%2FFU3Rz3k9k19tKrGLt2oLf0OC6GxBwok5c4swWnuPswleEQ%2FuDlRSQOYw3e0oQ78W5m%2FjpIID8x0cEBiCFL3UkwMycPAn%2FI1RJqdFHGEJLpxo7vNx81rVYF5oaMsXFBvKJ2arIQoh8Fa5bcluSJo7rGuC3Oi1JAP9kLe4EYRyTo%2BQDXM42dgDk2eCIVNkvXYghkxBJ1p2PxiyhA6b1hAY6tHffUAkoS08TFYkA4n28AWuWZPTFC7bmsuUVamhFIRoHjMI0zaZzrxl5g15ILzhBRHyfVb9lkDaSzDLmmKI5hyg7Zq3I%2FOOtbVKcT6Su0ZXuGqDO%2FShmuBZjXxciXjofZnaQnucRp0U1nNjMb9XbOk71rNSqbTWOyz14eZUb2lOjbNShYZTx5otarDWqlBIxDqiTu3KnAqho60NoArd%2FjThVhBxD5K30wURwLVgj0GE6HQ57qmDiCFZ%2B0vHh1us1czTslY7qQ9cGqAyfPxevdgRKCrXtqbXgkVFtd83ePRCjBYPN2ZGS2N2XrGhaQgpY%2BvQ2FCwiq7bjzGzRetrDGDMtOY1h0seez%2BdJ%2BaxHYmnPqzOazZnWzN%2FPLNZJeURhtsLj3WWyrj%2Ff4nlhHSVI9mfoGpDxnL6zxvLSZOv1i6TKsujHm4Z22ZNzyvMU4VlGcc40DJqWoMD37FlVIXMmqsOYBkLmbvEeceqPVVwbBzJ4v2ghlFv3u5zcXr25K24xHvqGF5vdmgvq%2Fd7sVYT13hPzlpZDH9JzxzAWl0%2FN9YOuZ2qzd7I%2F%2BtykyZkTSxrO3E36GqTPuRurLyVg9X5WYKC4JQGC5g9zH032tkDJLZVB4mxDRJ9UJDINnYNFsY2zuiZxHumEKapysGp0IKDjS11FPGJOVdVUwYI%2BYzmkO%2FiO%2B7nYAiZLlWVbK4f1MMwhjya9Us7j4ZxdryVZTEvqZpD1jBEU3F63jbnanz0ImVtMxOXBO5ZozP8oILA9rY9%2FcHcQTokSgQR43lWpFQwhHbiqiRno9qiZrPzMzmqPZgdW3ZMSAZf0Y3pDr7N6agh4NtUQ0SvztGrzDDxIRl7OWtusp6TD%2BNxlX7VySO17h7Z2jnbDLM8rlx9LopRiqhQZrdkdQMQofAtL%2F0EwxfIZCIvzzT3OMlVNyuPMYl4ZbGjgswr9Tiz7uY1nc1sFHQJdVC2KyzSiZlGeApWP35kBK7V8t9blwp6ETSfxlYAHw4b1LF9r%2F87EFFi02wmngLgpbRPmvI7%2FfkQgfVTNnBlTH8iFPOrkcY6ol5lLb4LiuHmpaNZaJhdYXg5Gk4ywkVLMesFezRQziTmiYB4Dj9k0zBhwhSCZUI9xOyCogZFq%2BgJce1xVaqPivjsqzX6FSUC0xWJ%2BaV9WwU7H1GBfPtu354f5O7scLKLV19E6zl76cr1DCTIu%2Faxt4oyV6R0jkIYbNo%2FzFEpEyfqtmeiy7Z%2BqP2dNm9eTmkUfEyWCxAnOTby3JfCnMkxn6KbrBYgaaUI0XCHTRQrUwqDQ4edvUtGbMeHHiZZvnKcLpD3HMMkkRhn8V7e0533NRr7EIO0MOPcG1V8lCxD8Fbcnr3VRFN%2BQ9ESkxTEacVPaPNOBhyGzO1gvVqwhiiOXxBeJRXFuJECwo%2FnK3gJ875fN4uKVHjaZKr7P7ivKopxbdadctnbPgaNKYuGT7vraa%2Fzn5INvfJNxZUjLvvsgRISDT6ATuCNxOCNllieA2fBe%2BDYuWvKbHs6xuRsOZM8OsWyKmC52Im0dyZdJhc9nRdUlcKEFPtv6gcGJRXsYyvYA2TrC3T0kfWbpiDNNDMOZPraq%2Bfu271Yi9XZO1Fvsm%2Fe8Qo9%2F5LKZ%2FbVjfJ2bBHvtiLR3YoucVt6W%2Bsze9y8lb8Uh7new7HYN6HjGzJWOtpM746VgoTKjPCwjLRkVrgjRgYJSa5J6XGJbtIHwhk9ybjMohIvP3z8Gc%2Bv2rK9OYvKA4WfMoe6rcwlZ7c0WajSm79m9bh29yXzsqvSLvWmO5T%2FHqSZehOnF2eZOR7Oqe7RGQ4cD3pSZ3jmmIZ58DtdOj2k19qpts5rc4rrin6we%2BhxBNfZdqnNnlxqQ%2Bjzvh5ycYiwVw%2FZanaVLhme1hme3paGqbdJNT4bA2bYYz0rjdF1o%2B05swRPeRq2NEUSz7LYvTWQu9DjmdbMsQxwpmN3upUxe%2F3yhDp7kMZuq2yKAxAmcKcQ7l6eFPt11QiMiy8r0dM1cMr2M%2FcHzuas%2B2U%2Fy%2BjABZ5fdT%2BLKhyck7xlsavtLPRy8%2B783AHZ%2FA8E%2Bv1%2F%3C%2Fdiagram%3E%3C%2Fmxfile%3E)
- google drive : [ts-fsrs-workflow.drawio](https://drive.google.com/file/d/1FLKjpt4T3Iis02vjoA10q7vxKCWwClfR/view?usp=sharing)
- github : [ts-fsrs-workflow.drawio](https://github.com/open-spaced-repetition/ts-fsrs/blob/7253365/ts-fsrs-workflow.drawio)

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ ts-fsrs 是一个基于TypeScript开发的[ES modules包](https://gist.github.co
...
...ApWP35kBK7V8t9blwp6ETSfxlYAHw4b1LF9r%2F87EFFi02wmngLgpbRPmvI7%2FfkQgfVTNnBlTH8iFPOrkcY6ol5lLb4LiuHmpaNZaJhdYXg5Gk4ywkVLMesFezRQziTmiYB4Dj9k0zBhwhSCZUI9xOyCogZFq%2BgJce1xVaqPivjsqzX6FSUC0xWJ%2BaV9WwU7H1GBfPtu354f5O7scLKLV19E6zl76cr1DCTIu%2Faxt4oyV6R0jkIYbNo%2FzFEpEyfqtmeiy7Z%2BqP2dNm9eTmkUfEyWCxAnOTby3JfCnMkxn6KbrBYgaaUI0XCHTRQrUwqDQ4edvUtGbMeHHiZZvnKcLpD3HMMkkRhn8V7e0533NRr7EIO0MOPcG1V8lCxD8Fbcnr3VRFN%2BQ9ESkxTEacVPaPNOBhyGzO1gvVqwhiiOXxBeJRXFuJECwo%2FnK3gJ875fN4uKVHjaZKr7P7ivKopxbdadctnbPgaNKYuGT7vraa%2Fzn5INvfJNxZUjLvvsgRISDT6ATuCNxOCNllieA2fBe%2BDYuWvKbHs6xuRsOZM8OsWyKmC52Im0dyZdJhc9nRdUlcKEFPtv6gcGJRXsYyvYA2TrC3T0kfWbpiDNNDMOZPraq%2Bfu271Yi9XZO1Fvsm%2Fe8Qo9%2F5LKZ%2FbVjfJ2bBHvtiLR3YoucVt6W%2Bsze9y8lb8Uh7new7HYN6HjGzJWOtpM746VgoTKjPCwjLRkVrgjRgYJSa5J6XGJbtIHwhk9ybjMohIvP3z8Gc%2Bv2rK9OYvKA4WfMoe6rcwlZ7c0WajSm79m9bh29yXzsqvSLvWmO5T%2FHqSZehOnF2eZOR7Oqe7RGQ4cD3pSZ3jmmIZ58DtdOj2k19qpts5rc4rrin6we%2BhxBNfZdqnNnlxqQ%2Bjzvh5ycYiwVw%2FZanaVLhme1hme3paGqbdJNT4bA2bYYz0rjdF1o%2B05swRPeRq2NEUSz7LYvTWQu9DjmdbMsQxwpmN3upUxe%2F3yhDp7kMZuq2yKAxAmcKcQ7l6eFPt11QiMiy8r0dM1cMr2M%2FcHzuas%2B2U%2Fy%2BjABZ5fdT%2BLKhyck7xlsavtLPRy8%2B783AHZ%2FA8E%2Bv1%2F%3C%2Fdiagram%3E%3C%2Fmxfile%3E)来获取ts-fsrs的工作流信息。
```
Comment (L-M-Sherlock) [inline README_CN.md]: 这个链接怎么这么长

Comment (ishiko732) [reply README_CN.md]: > 这个链接怎么这么长

因为draw.io生成的外部链接包含了数据信息所以就那么长了...我试了用短链接但是会打不开

Comment (ishiko732) [reply README_CN.md]: 直接指引repo的[ts-fsrs-workflow.drawio](https://github.com/open-spaced-repetition/ts-fsrs/blob/7253365/ts-fsrs-workflow.drawio)文件吧

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Pull request merged**